bgpd/bgp_attr.h

```diff
@@ -96,6 +98,18 @@ struct overlay_index {
 	union gw_addr gw_ip;
 };
 
+enum pta_type {
+	PMSI_TNLTYPE_NO_INFO,
+	PMSI_TNLTYPE_RSVP_TE_P2MP,
```

Comment:
Hi @pguibert6WIND - this is probably a recent comment, otherwise my apologies for missing it. Yes, I can certainly switch to using the lookup_msg() API which you have referred to below too. I shall also define the first enum explicitly to 0. It seems to me an enum is appropriate here since only a single well-defined value is possible for the tunnel type.